edx_sga/sga.py

```diff
@@ -902,13 +902,15 @@ def get_student_data():
                     'may_grade': instructor or not approved,
                     'annotated': force_text(state.get("annotated_filename", '')),
                     'comment': force_text(state.get("comment", '')),
-                    'finalized': is_finalized_submission(submission_data=submission)
+                    'finalized': is_finalized_submission(submission_data=submission),
+                    'team': self.get_team_name(student_module.student.username)
```

Comment:
Always leave a trailing `,` in every dict element. In the future if you modified the dict to add another element, then only one line of code will be affected therefore it is easier to review.

Reply:
check